### PR TITLE
Cmac/bfd 691 update coverage carin mappings

### PR DESCRIPTION
**Pull request by mackec1**
### Change Details

No changes in CARIN mappings; Parts A, B, C, D conformance check results in zero errors & zero warnings.

- apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java
- apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

### Acceptance Validation

Verify that Coverage implementation mappings conform to FIHR R4 using CARIN IG. This can be cross-checked vs 
- BFD-EOB CARIN Discovery spreadsheet, Coverage tab (Confluence, google doc; in process of being versioned so unable to provide link)
- FIHR V4 Conformance tool (https://www.hl7.org/fhir/downloads.html)

### Feedback Requested

Code smells and/or poor structure of code.

### External References

- [BFD-691](https://jira.cms.gov/browse/BFD-691)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [X] new data stored or transmitted

V2 CARIN mappings

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by cbrunefearless — DISMISSED** [review]
LGTM

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java [hunk 1]
```diff
@@ -40,14 +40,50 @@ public void setup() {
             .map(r -> (Beneficiary) r)
             .findFirst()
             .get();
-    beneficiary.setLastUpdated(new Date());
+    // beneficiary.setLastUpdated(new Date());
```
Comment (Dranith) [inline apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java:43]: I'd remove this if it isn't necessary.  This _should_ play into the end result though I would expect?

**Review by Dranith — CHANGES_REQUESTED** [review]
Looks good, just one question about a commented out piece of code.

**Review by Dranith — APPROVED** [review]
Looks good.

**Review by cbrunefearless — APPROVED** [review]
lgtm

**Pull request merged**